test/CadlRanchProjects/type/union/src/Generated/Models/ModelWithNamedUnionProperty.cs

```diff
@@ -32,6 +32,17 @@ public ModelWithNamedUnionProperty(BinaryData namedUnion)
         /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
         /// </para>
         /// <para>
+        /// <remarks>
+        /// The following types are supported by this property:
+        /// <list type="bullet">
+        /// <item>
+        /// <c>Model1</c>
```

Comment:
These should be cref's if they are real classes we know about.  String literals would fall outside of that.